java/ql/lib/ext/java.net.model.yml

```diff
@@ -21,6 +22,8 @@ extensions:
       pack: codeql/java-all
       extensible: summaryModel
     data:
+      - ["java.net", "URI", True, "resolve", "(URI)", "", "Argument[0]" "ReturnValue", "taint", "ai-generated"]
+      - ["java.net", "InetAddress", True, "getAllByName", "(String)", "", "Argument[0]" "ReturnValue.ArrayElement", "taint", "ai-generated"]
       - ["java.net", "URI", False, "URI", "(String)", "", "Argument[0]", "Argument[-1]", "taint", "manual"]
```

Comment:
Small nitpick: should the alphabetical ordering of the models be maintained for better human readability of the yml files? I've been making an effort to do so whenever I insert models, but maybe that's been a waste of my time. 😅